types/node/v10/package.json

```diff
@@ -6,6 +6,11 @@
             "*": [
                 "ts3.1/*"
             ]
+        },
+        ">=3.2.0-0": {
```

Comment:
I don't think splitting typings into even more versions is a good idea, bumping up 3.1 to 3.2 should be fine.

Reply:
@SimonSchick thanks for the feedback. I've added a new commit to consolidate the versions.